Mathlib/NumberTheory/Bernoulli.lean

```diff
@@ -23,7 +23,7 @@ theory of modular forms). For example, if $1 \leq n$ is even then
 
 $$\zeta(2n)=\sum_{t\geq1}t^{-2n}=(-1)^{n+1}\frac{(2\pi)^{2n}B_{2n}}{2(2n)!}.$$
 
-Note however that this result is not yet formalised in Lean.
+This result is formalised in Lean: `riemannZeta_two_mul_nat`
```

Comment:
```suggestion
This result is formalised in Lean: `riemannZeta_two_mul_nat`.
```

Also, I cannot add a suggestion via this interface, but please delete "is even" from the previous sentence as it's unnecessary (the formula is already expressed in terms of 2n, not n).

Thanks!
bors d+